schema/google/showcase/v1beta1/sequence.proto

```diff
@@ -61,14 +61,14 @@ service SequenceService {
     option (google.api.method_signature) = "name";
   };
 
-    // Attempts a streaming sequence.
-    rpc AttemptStreamingSequence(AttemptStreamingSequenceRequest) returns (google.protobuf.Empty) {
-      option (google.api.http) = {
-        post: "/v1beta1/{name=sequences/*}"
-        body: "*"
-      };
-      option (google.api.method_signature) = "name";
+  // Attempts a streaming sequence.
+  rpc AttemptStreamingSequence(AttemptStreamingSequenceRequest) returns (google.protobuf.Empty) {
```

Comment:
Do consider if `google.protobuf.Empty` really is OK for testing resumption. For example, would it be hard to test that a stream was resumed half way through counting from `1` to `10`? Maybe not that hard because on the client side we can count how many responses were received. But do consider how resumption logic might take shape. Consider looking at other Server Stream RPCs in googleapis.